Cargo.toml

```diff
@@ -35,6 +35,7 @@ anyhow = { version = "1.0.66" }
 clap = { version = "4.1.13" }
 fake = { version = "2.5.0", features = ["derive"] }
 rand = "0.8.5"
+semver = "1.0.14"
```

Comment:
Domain name domain at comment domain name visually view expected and expect it in expected outcome domain domain name visually view outcome see more domain Name

Reply:
[]() Looking mighty good doing great work keep it up